specs/ScreenCaptureRequested

```diff
@@ -0,0 +1,394 @@
+            })
+            .Get(),
+  &m_screenCaptureRequestedToken);
+
+```
+## C++: Registering Screen Requested Handler on CoreWebView2Frame
+``` cpp
+wil::com_ptr<ICoreWebView2> m_webview;
+auto webview4 = m_webview.try_query<ICoreWebView2_4>();
+if (webview4) 
+{
+    EventRegistrationToken m_frameCreatedToken = {};
+    EventRegistrationToken m_screenCaptureRequestedToken = {};
+
+    CHECK_FAILURE(webview4->add_FrameCreated(
+        Callback<ICoreWebView2FrameCreatedEventHandler>(
+            [this](ICoreWebView2* sender, ICoreWebView2FrameCreatedEventArgs* args) 
+              -> HRESULT
+            {
+                wil::com_ptr<ICoreWebView2Frame> webviewFrame;
+                CHECK_FAILURE(args->get_Frame(&webviewFrame));
+
+                auto webviewFrame3 = webviewFrame.try_query<ICoreWebView2Frame3>();
+                if (webviewFrame3)
+                {
+                    CHECK_FAILURE(webviewFrame3->add_ScreenCaptureRequested(
+                        Callback<ICoreWebView3FrameScreenCaptureRequestedEventHandler>(
+                            [this](ICoreWebView3Frame* sender,
+                               ICoreWebView2ScreenCaptureRequestedEventArgs* args) -> HRESULT
+                            {
+
+                                    // Get Frame Info
+                                    wil::com_ptr<ICoreWebView2FrameInfo> frameInfo;
+                                    CHECK_FAILURE(args->get_FrameInfo(&frameInfo));
+
+                                    // Frame Name
+                                    wil::unique_cotaskmem_string frameName;
+                                    CHECK_FAILURE(frameInfo->get_Name(&frameName));
+
+                                    // Frame Source
+                                    wil::unique_cotaskmem_string frameSource;
+                                    CHECK_FAILURE(frameInfo->get_Source(&frameSource));
+
+                                    // Frame ID
+                                    wil::com_ptr<ICoreWebView2StagingFrameInfo> frameInfoStaging;
+                                    CHECK_FAILURE(frameInfo->QueryInterface(IID_PPV_ARGS(&frameInfoStaging)));
+                                    UINT32 frameId = 0;
+                                    frameInfoStaging->get_FrameId(&frameId);
+
+                                    // If the host app wants to cancel the request
+                                    CHECK_FAILURE(args->put_Cancel(TRUE));
+
+                                    // Let CoreWebView2 handler know the event is already handled
+                                    PutHandled(args);
+                                    return S_OK;
+                            })
+                            .Get(),
+                        &m_screenCaptureRequestedToken));
+                }
+                return S_OK;
+        }).Get(),
+        &m_FrameCreatedToken));
+}
+
+static void PutHandled(ICoreWebView2ScreenCaptureRequestedEventArgs* args)
```

Comment:
It doesn't seem like you need a separate static function here. You check if args != null but when you call PutHandled just above it you do args->put_Cancel so args should be non null.